src/components/ListDeckInfo/DeckList.js

```diff
@@ -6,8 +6,8 @@ export default function DecksList({decks}) {
   return (
     <div className={styles.main_content}>
       {decks.length > 0 ? (
-        decks.map(deck => {
-          return <DeckCard deck={deck} />;
+        decks.map((deck, index) => {
```

Comment:
Please use the deck.deck_id for the key prop. We don't want to use index for map keys